src/mongocxx/test_util/client_helpers.hh

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016 MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <cstdint>
+
+#include <mongocxx/config/private/prelude.hh>
+
+namespace mongocxx {
+MONGOCXX_INLINE_NAMESPACE_BEGIN
+
+class client;
+
+namespace test_util {
+
+///
```

Comment:
Hmm.  I investigated, and confirmed indeed that other header comments for private code don't use `@param` etc, but they do use doxygen-style triple-comments.  Changed in fixup commits.

It'd be worth thinking at some point about a way to make the header-comments-for-private-code situation make more sense going forward.